components/centraldashboard/public/components/localization-mixin.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line max-len
+import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
+import {languages} from '../assets/i18n/languages.json';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {PolymerElement} from '@polymer/polymer';
+
+// eslint-disable-next-line max-len
+export default class extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
```

Comment:
I think defining the mixin this way breaks the inheritance chaining. What happens if you take a superclass argument like the utilities mixin does and pop that in stead of `PolymerElement`?

Reply:
Modified to use superclass argument instead